oidc_cli/oidc_impl/cache/cache.go

```diff
@@ -109,3 +129,31 @@ func (c *Cache) readFromStorage(ctx context.Context) (*client.Token, error) {
 	}
 	return cachedToken, nil
 }
+
+func compressToken(token string) (string, error) {
+	var buf bytes.Buffer
+	gz := gzip.NewWriter(&buf)
+	if _, err := gz.Write([]byte(token)); err != nil {
+		return "", fmt.Errorf("failed to write to gzip: %w", err)
+	}
+	if err := gz.Close(); err != nil {
+		return "", fmt.Errorf("failed to close gzip: %w", err)
+	}
```

Comment:
I think you have to close these. Closing does some kind of flushing of bytes I think. Here's my defer statement:

https://github.com/chanzuckerberg/scruffy/blob/ed5a136fbce6357a473a86047179215d4fd72f17/pkg/plugin/start.go#L117

Reply:
o duh, I totally missed it. ignore